app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :posts
```

Comment:
- [ ] It should have many Events
- [ ] Make sure to specify the foreign_key as creator_id